benchmarks/decomposition.py

```diff
@@ -1,27 +1,32 @@
-from sklearn.decomposition import PCA, DictionaryLearning
+from sklearn.decomposition import (
+    PCA,
+    DictionaryLearning,
+    MiniBatchDictionaryLearning,
+)
 
 from .common import Benchmark
-from .datasets import _mnist_dataset
+from .datasets import _mnist_dataset, _decomposition_dataset
 
 
 class PCA_bench(Benchmark):
     """
     Benchmarks for PCA.
     """
+
     # params = (svd_solver)
     param_names = ['params']
-    params = ([('full',),
-               ('arpack',),
-               ('randomized',)],)
+    params = ([('full', ), ('arpack', ), ('randomized', )], )
```

Comment:
doesn't it overwrite the n_jobs params? This (and other) benches need to take n_jobs into account